### PR TITLE
Dynamically try and set MTU

### PR DESCRIPTION
**Pull request by cloudnautique**
We are not accounting for IPSec tunnel or IP headers from our network container in our container MTU sizes. This causes UDP performance to plummet. 

This PR looks at the default gateways interface.

### CODE REVIEW

File: scripts/lib/net-util.sh [hunk 1]
```diff
@@ -20,6 +20,15 @@ wait_for_dev()
     fi
 }
 
+calc_mtu_size()
+{
+   iface=$(ip route show default 0.0.0.0/0|cut -d' ' -f5)
+   iface_mtu=$(ip link show dev eth0 | grep mtu | awk '{print $5}')
+
+   # remove enough for IPSEC and IP headers from network agent.
+   echo $(($iface_mtu-98))
```
Comment (ibuildthecloud) [inline scripts/lib/net-util.sh]: don't hate me.... but 4 spaces for indent.

Comment (ibuildthecloud) [inline scripts/lib/net-util.sh]: don't hate me, but 4 spaces for indent.

File: scripts/lib/net-util.sh [hunk 2]
```diff
@@ -39,6 +48,10 @@ while [ "$#" -gt 0 ]; do
         shift 1
         DEV=$1
         ;;
+	-m)
+        shift 1
+		MTU=${1}
```
Comment (ibuildthecloud) [inline scripts/lib/net-util.sh]: and no tabs

File: scripts/lib/net-util.sh (hunk 1 above)
Comment (vincent99) [inline scripts/lib/net-util.sh]: 98<sup>[citation needed]</sup>..  More info on where that number came from would be nice.  e.g. is this also enough for IPv6?

File: events/start_handler.go [hunk 3]
```diff
@@ -168,6 +170,7 @@ func (h *StartHandler) Handle(event *docker.APIEvents) error {
 		}
 
 		return err
+
```
Comment (ibuildthecloud) [inline events/start_handler.go]: extra whitespace

**Pull request merged**